### ISSUE TITLE
Radio error popup gets in the way

### ISSUE THREAD

**Issue by madprops** [audacious-media-player/audacious#1509]
I have a playlist with some radio stations.

When I double click an item, if it fails to load it shows a popup.

The thing is that popup takes all the focus.

I barely leaves me time to dismiss it to do another action before another popup popups up since it tried to use the next radio station (that also failed).

I think it shouldn't show a popup, there must be something less intrusive.

**Comment by WildPenquin** [comment]
> The thing is that popup takes all the focus.

I agree, this is a major issue when playing music in the background while doing something else such as playing a game. An error pop up will minimize the game window and steal all focus.


IMHO no application which is run in the backgroud should ever steal focus.

There needs to be some less intrusive way to present errors to the user.

**Comment by MikeSS8** [comment]
A notification (aka "toast") seems like the "less intrusive" way for a background program to communicate something to the user.  The GTK UI could use libnotify, but I'm not sure there's a standard way to do toasts in Qt.
